src/test/java/string_calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,33 @@
+package string_calculator;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+public class StringCalculatorTest {
+    @ParameterizedTest(name = "{0}이 주어지면 결과는 {1}이다.")
+    @CsvSource(value = {"2 + 3 * 4 / 2:10", "1 + 3:4", "5 - 2 / 3:1", "6 * 2 * 4:48", "3 + 3 + 3 + 1 / 3:3"}, delimiter = ':')
```

Comment:
가독성을 위해 개행을 추가하면 어떨까요?
```suggestion
    @CsvSource(value = {
            "2 + 3 * 4 / 2:10", 
            "1 + 3:4", 
            "5 - 2 / 3:1", 
            "6 * 2 * 4:48", 
            "3 + 3 + 3 + 1 / 3:3"
    }, delimiter = ':')
```

Reply:
테스트의 입력이라 신경을 덜 썼네요.
감사합니다.